src/bench/addrman.cpp

```diff
@@ -74,14 +78,14 @@ static void AddrManAdd(benchmark::Bench& bench)
     CreateAddresses();
 
     bench.run([&] {
-        AddrMan addrman{/* asmap */ std::vector<bool>(), /* deterministic */ false, /* consistency_check_ratio */ 0};
+        AddrMan addrman{EMPTY_ASMAP, !DETERMINISTIC, ADDRMAN_CONSISTENCY_CHECK_RATIO};
```

Comment:
why the `!`?

Reply:
Because `false` was passed in the original code, denoting "not deterministic".

Reply:
It appears a bit confusing to me to set `DETERMINISTIC=true` and get non-determinism

Reply:
I think it is ok, but if it is confusing somebody then it is not good. Should I go back to `/*deterministic=*/false`?

Reply:
no opinion on that, but if you want to keep `DETERMINISTIC`, it might be better set to false

Reply:
Reverted to `/*deterministic=*/false`.